db/migrate/20160412223500_add_institution_name_full_to_site.rb

```diff
@@ -0,0 +1,5 @@
+class AddInstitutionNameFullToSite < ActiveRecord::Migration
+  def change
+    add_column :sites, :institution_name_full, :string
```

Comment:
Okay. Would I just edit the text of the migration to combine the two `add_column` statements?